src/index.js

```diff
@@ -19,17 +19,49 @@
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  *
  */
-import { getParserManager } from './parsers/parserManager.js'
-import TimezoneAdapter from './timezones/timezoneAdapter.js'
-import { getTimezoneManager } from './timezones/timezoneManager.js'
+import ParserManager, { getParserManager } from './parsers/parserManager.js'
+import { getTimezoneManager, TimezoneManager } from './timezones/timezoneManager.js'
 import { v4 as uuid } from 'uuid'
-import DateTimeValue from './values/dateTimeValue.js'
 import { dateFactory } from './factories/dateFactory.js'
-import CalendarComponent from './components/calendarComponent.js'
-import EventComponent from './components/root/eventComponent.js'
-import RecurrenceManager from './recurrence/recurrenceManager.js'
-import FreeBusyComponent from './components/root/freeBusyComponent.js'
 import ICAL from 'ical.js'
+import AbstractComponent from './components/abstractComponent'
```

Comment:
All of these are not very readable. Why not import & export everything while grouping by folder, from `components/index.js`, `errors/index.js`, etc. instead?

Reply:
That is a good point. I'll refactor this.